doc/source/index.rst

```diff
@@ -4,74 +4,67 @@
 Percona XtraDB Cluster |version| Documentation
 ==============================================
 
-|PXC|_ is a database clustering solution for MySQL.
-It ensures high availability, prevents downtime and data loss,
-and provides linear scalability for a growing environment.
+|PXC|_ is a database clustering solution for MySQL.  It ensures high
+availability, prevents downtime and data loss, and provides linear scalability
+for a growing environment.
 
-Features of |PXC| include:
+.. rubric:: Features of |PXC|
 
-* **Synchronous replication**
-
-  Data is written to all nodes simultaneously,
-  or not written at all if it fails even on a single node.
-
-* **Multi-master replication**
-
-  Any node can trigger a data update.
-
-* **True parallel replication**
-
-  Multiple threads on slave performing replication on row level.
-
-* **Automatic node provisioning**
-
-  You simply add a node and it automatically syncs.
-
-* **Data consistency**
-
-  No more unsynchronized nodes.
-
-* **PXC Strict Mode**
-
-  Avoids the use of experimental and unsupported features.
-
-* **Configuration script for ProxySQL**
-
-  Percona provides a ProxySQL package with the ``proxysql-admin`` tool
-  that automatically configures |PXC| nodes.
-
-* **Automatic configuration of SSL encryption**
-
-  |PXC| includes the ``pxc-encrypt-cluster-traffic`` variable
-  that enables automatic configuration of SSL encrytion.
-
-* **Optimized Performance**
-
-  |PXC| performance is optimized to scale with a growing production workload.
-
-  For more information, see the following blog posts:
-
-  * `How We Made Percona XtraDB Cluster Scale
-    <https://www.percona.com/blog/2017/04/19/how-we-made-percona-xtradb-cluster-scale/>`_
-
-  * `Performance improvements in Percona XtraDB Cluster 5.7.17-29.20
-    <https://www.percona.com/blog/2017/04/19/performance-improvements-percona-xtradb-cluster-5-7-17/>`_
-
-|PXC| is fully compatible with `MySQL Server Community Edition <MySQL>`_,
+.. admonition:: |author|
+   
+   TODO: For all features, provide links to relevant sections in documentation.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
```

Comment:
We should mention data at rest encryption in the feature list, and streaming replication/transaction support, those are kind of the two big deals from PS 8.0 + Galera 4.